app/views/index.xml

```diff
@@ -1,5 +1,69 @@
 <Alloy>
 	<Window class="container">
-		<Label id="label" onClick="doClick">Hello, World</Label>
+		<View id="header">
+			<ImageView id="headerLeft" image="/global/blind2.png"></ImageView>
+			<Label id="headerText">Victim Services</Label>
+			<ImageView id="headerRight" image="/global/add126.png"></ImageView>
+		</View>
+		
+		<!--dropshadow table-->
+		<TableView id="dropShadowMenu">
```

Comment:
Not sure if this is the best way to accomplish the drop shadow effect. When the menu actually goes off the screen and scrolling is enabled, the drop shadows will scroll by themselves and aren't attached to the actual menu items. Can the dropshadow view be placed in the "menu" tableview rows?